test/users/post.test.js

```diff
@@ -273,13 +274,47 @@ describe('POST /users/sessions', () => {
         expect(response.status).toBe(200);
       }));
 
-    test('Check that there is a token', () =>
+    it('Check that there is a token', () =>
```

Comment:
it y test son lo mismo. Uno es un alias del otro

Reply:
Si, lo cambié sólo para ser consistente